### PR TITLE
Server Updates

### PR DESCRIPTION
**Pull request by kevinshare**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  "extends": "airbnb",
+  "plugins": ["jest"],
+  "env": {
+    "jest/globals": true
+  }
+};
```
Comment (ZeMunchkin) [inline .eslintrc.js]: new line

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,2 @@
 node_modules
 .env
-router.js
-server.test.js
```
Comment (ZeMunchkin) [inline .gitignore:4]: new line

File: public/style.css [hunk 3]
```diff
@@ -1,30 +1,38 @@
...
   display: flex;
-  height: 100%;
+  height: 50%;
+  margin-top: 40%;
+  width: 25%;
   flex-flow: row wrap;
   display: flex;
-  align-items: stretch;
   flex-direction: column;
 }
```
Comment (ZeMunchkin) [inline public/style.css:38]: new line

File: router.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  const url = `http://foodigoreviews.us-west-1.elasticbeanstalk.com/restaurants/${path}/reviews`;
+  request(url).pipe(res);
+});
+router.put('/restaurants/:restaurantId/reviews/:reviewId', (req, res) => {
+  const path = req.params.restaurantId;
+  const review = req.params.reviewId;
+  const url = `http://foodigoreviews.us-west-1.elasticbeanstalk.com/restaurants/${path}/reviews/${review}`;
+  request(url).pipe(res);
+});
+router.post('/restaurants/:restaurantId/reviews/:reviewId', (req, res) => {
```
Comment (ZeMunchkin) [inline router.js]: the post route doesn't have a review id. You might not have a functional post request with this route

**Review by ZeMunchkin — CHANGES_REQUESTED** [review]
There's a route that I think won't work, and you should double check that you have all of your linter errors worked out

**Review by mattman240 — APPROVED** [review]
Looks fine

**Pull request merged**